mnefun/_utils.py

```diff
@@ -312,3 +313,123 @@ def repeat_coreg(subject, subjects_dir=None, subjects_dir_old=None,
+    just processed differently. Given a surrogate created using the old
+    template, this function will create an equivalent one for the new
+    template. It operates in-place by first backing up (renaming) the MRI
+    directory for the subject, copying the ``-trans.fif`` file to that
+    directory, and then creating the new surrogate and overwriting the old
+    trans file.
+    """
+    # load morph params
+    subjects_dir = mne.utils.get_subjects_dir(subjects_dir)
+    config = mne.coreg.read_mri_cfg(subject, subjects_dir)
+    n_params = config.pop('n_params')
+    subject_from = config['subject_from']
+    if subject_from not in ('ANTS3-0Months3T', 'ANTS6-0Months3T',
+                            'ANTS12-0Months3T'):
+        raise RuntimeError('Cannot convert subject that used '
+                           f'{repr(subject_from)} as a surrogate')
+    age = int(subject_from.split('-')[0].split('S')[-1])
+    assert n_params in (3, 1), n_params
+    out_dir = op.join(subjects_dir, subject)
+    backup_dir = op.join(subjects_dir, subject + '_old')
+    if not isinstance(trans, (str, os.PathLike)):
+        raise TypeError(f'trans must be path-like, got {type(trans)}')
+    assert isinstance(trans, str)
+    trans, trans_fname = mne.transforms._get_trans(trans, 'head', 'mri')
+    if op.exists(backup_dir):
+        raise RuntimeError(f'Backup dir {backup_dir} must not already exist')
+    backup_trans = op.join(out_dir, op.basename(trans_fname))
+    if op.exists(backup_trans):
+        raise RuntimeError(f'Backup trans location {backup_trans} must not '
+                           'exist')
+    from_dir = op.join(subjects_dir, subject_from)
+    if not op.isdir(from_dir):
+        raise RuntimeError(f'Template MRI directory not found: {from_dir}')
+    bem_path = op.join(
+        from_dir, 'bem', f'{subject_from}-5120-5120-5120-bem-sol.fif')
+    if not op.isfile(bem_path):
+        raise RuntimeError(f'{subject_from} in {repr(subjects_dir)} does not '
+                           'appear to be a new-style template, consider '
+                           'running:\n\n'
+                           'import shutil, mne\n'
+                           f'shutil.rmtree({repr(from_dir)})\n'
+                           f'mne.datasets.fetch_infant_template(\'{age}mo\''
+                           f', subjects_dir={repr(subjects_dir)}'
+                           ', verbose=True)\n')
+    shutil.move(trans_fname, backup_trans)
+    shutil.move(out_dir, backup_dir)
+    print('Rescaling MRI (will be slow)...')
+    mne.coreg.scale_mri(subject_to=subject, subjects_dir=subjects_dir,
+                        labels=False, annot=False, overwrite=False,
+                        **config)
+    bem_path = op.join(
+        out_dir, 'bem', f'{subject}-5120-5120-5120-bem-sol.fif')
+    sol = mne.make_bem_solution(mne.read_bem_surfaces(bem_path[:-8] + '.fif'))
+    mne.write_bem_solution(bem_path, sol)
```

Comment:
@nordme @mdclarke I think I fixed the no-bem-sol issue, feel free to try it if you haven't converted all subjects yet.

If you have converted all the ones you want to and things seem to be working I'll go ahead and merge!